lib/program.js

```diff
@@ -142,14 +142,14 @@ var Program = Base.extend({
         });
     },
 
-    // returns a list of the sinks of this program, formatted for use by the app
-    get_client_sinks: function() {
+    // returns a list of the views of this program, formatted for use by the app
+    get_client_views: function() {
```

Comment:
I think we should just use "view" instead of "clientview" as the proc name and call this simply `get_views()`.